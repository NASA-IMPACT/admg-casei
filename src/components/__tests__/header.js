import React from "react"
import renderer from "react-test-renderer"
import Header from "../header"

jest.mock("../image.js", () => {
  const mockImage = () => <img />
  return mockImage
})

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Header
          siteTitle="Site title from props"
          shortname="Shortname from props"
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
