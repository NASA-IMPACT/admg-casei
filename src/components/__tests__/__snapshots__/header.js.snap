// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "space-between",
        "margin": "0 auto",
        "maxWidth": 1280,
        "padding": "2rem 5rem",
      }
    }
  >
    <div
      style={
        Object {
          "fontFamily": "'Varela Round',sans-serif",
          "fontSize": "1.5rem",
          "lineHeight": "1.5rem",
          "margin": 0,
          "zIndex": 100,
        }
      }
    >
      <a
        href="/"
        style={
          Object {
            "columnGap": "0.25rem",
            "display": "grid",
            "gridTemplateColumns": "3rem auto",
            "gridTemplateRows": "1.5rem 1.5rem",
            "textDecoration": "none",
          }
        }
      >
        <div
          style={
            Object {
              "gridArea": "1 / 1 / 3 / 2",
            }
          }
        >
          <img />
        </div>
        NASA
        <b>
          Site title from props
        </b>
      </a>
    </div>
    <div
      style={
        Object {
          "display": "flex",
        }
      }
    />
  </div>
</header>
`;
